src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstanceFactory.cs

```diff
@@ -180,11 +180,13 @@ private async Task UpdateCurrentlyRunningInstanceAsync(ImmutableHashSet<string>
                 var majorVersion = int.Parse(instanceVersion.Substring(0, instanceVersion.IndexOf('.')));
                 hostProcess = StartNewVisualStudioProcess(installationPath, majorVersion, isUsingLspEditor);
 
+                /*
                 var procDumpInfo = ProcDumpInfo.ReadFromEnvironment();
                 if (procDumpInfo != null)
                 {
                     ProcDumpUtil.AttachProcDump(procDumpInfo.Value, hostProcess.Id);
                 }
+                */
```

Comment:
📝 Might as well delete this outright (we aren't going to ever use it)

Reply:
Properly deleted now.